node/commons/models/VtexSeller.ts

```diff
@@ -26,7 +26,8 @@ class VtexSeller {
     sellerId: string,
     price: number,
     oldPrice: number,
-    installment: { value: number; count: number }
+    installment: { value: number; count: number },
+    tax?: number,
```

Comment:
This class is no longer used, right? You changed it to use the already existing types for Seller and CommertialOffer,

Reply:
Yeah, good point, it's not being used anywhere anymore. Imma delete it in the next commit.